### PR TITLE
[10.0] MIG mrp repair discount

### PR DESCRIPTION
**Pull request by hurrinico**
Refactoring of https://github.com/OCA/manufacture/pull/180/commits/255e2ce2445b7ea442013b2f6588f6484cda656e

### CODE REVIEW

**Comment by hurrinico** [comment]
runbot and travis are bot green @eLBati @tafaRU

File: mrp_repair_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+This module extends the functionality of mrp repair adding
+new field discount on operations lines
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/129/8.0
```
Comment (tafaRU) [inline mrp_repair_discount/README.rst]: change branch to 10.0

File: mrp_repair_discount/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015 Nicola Malcontenti - Agile Business Group
```
Comment (tafaRU) [inline mrp_repair_discount/__init__.py]: Replace © by Copyright, here and in all headers.

See https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L2 as reference.

File: mrp_repair_discount/models/mrp_repair.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# © 2015 Nicola Malcontenti - Agile Business Group
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
```
Comment (tafaRU) [inline mrp_repair_discount/models/mrp_repair.py]: Replace everywhere `openerp` by `odoo`.

File: mrp_repair_discount/views/mrp_repair_view.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (tafaRU) [inline mrp_repair_discount/views/mrp_repair_view.xml]: Replace `openerp` by `odoo` and remove `data` tag.

**Review by tafaRU — CHANGES_REQUESTED** [review]

**Comment by hurrinico** [comment]
@tafaRU done

File: mrp_repair_discount/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (eLBati) [inline mrp_repair_discount/__openerp__.py]: @hurrinico rename to ```__manifest__.py```

**Review by eLBati — CHANGES_REQUESTED** [review]
1 request, then :+1: , tested

**Comment by tafaRU** [comment]
@hurrinico, thanks for the changes!
Please ping me once https://github.com/OCA/manufacture/pull/222#discussion_r137458104 will be addressed.

**Comment by hurrinico** [comment]
@tafaRU @eLBati fixed and everything is green

**Review by eLBati — APPROVED** [review]

File: mrp_repair_discount/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Nicola Malcontenti - Agile Business Group
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "MRP Repair Discount",
+    "version": "10.0.1.0.0",
+    "category": "Manufactoring",
```
Comment (tafaRU) [inline mrp_repair_discount/__manifest__.py]: s/Manufactoring/Manufacturing

**Comment by hurrinico** [comment]
@tafaRU done

**Review by tafaRU — APPROVED** [review]

**Comment by tafaRU** [comment]
@hurrinico, thanks!

**Comment by tafaRU** [comment]
@pedrobaeza, I think this PR could be merged. Could you please check it?
Thank you in advance.

**Pull request merged**